fullstack_template/server/server.py

```diff
@@ -31,25 +33,25 @@ def identity(payload):
 
 @app.route('/register',methods=['POST'])
 def register():
-    if request.method == 'POST':
-        request_data = request.get_json()
-        data = User(first_name=request_data["first_name"],
-                    last_name=request_data["last_name"], 
-                    email=request_data["email"],
-                    password=hash_password(request_data["password"]),
-                    major=None)
-        user = User.query.filter_by(email=data.email).first()
-        if user is not None: 
-            return "unsucessful register"
-        try:     
-            db.session.add(data)
-            db.session.commit()        
-            db.session.close()
-        except:
-            db.session.rollback()
-        return "successful register"
-    else:
-        return "unsucessful register"
+    request_data = request.get_json()
+    data = User(first_name=request_data["first_name"],
+                last_name=request_data["last_name"], 
+                email=request_data["email"],
+                password=hash_password(request_data["password"]),
+                major=None)
+        
+    #email validation
+    user = User.query.filter_by(email=data.email).first()
+    if user is not None: 
+        return jsonify(message="Email already registered"), 400
```

Comment:
Does this 400 here return a status of 400?

Reply:
Yep